apps/web/pages/my-account/my-orders.vue

```diff
@@ -1,89 +1,133 @@
 <template>
```

Comment:
we might want to have a loading spinner or disable the pagination while the data is loading. The calls take some time and if you are too fast you might get a call thats not for the page :D